content/millennium/dstu2/scheduling/slot.md

```diff
@@ -52,25 +53,28 @@ Search for Schedules that meet supplied query parameters:
  Name             | Required?                                                        | Type           | Description
 ------------------|------------------------------------------------------------------|----------------|------------------------------------------------------------------------------------
 `_id`             | This, or `slot-type`                                             | [`token`]      | The logical resource id associated with the resource.
-`slot-type`       | Yes when using `schedule.actor` and\or `-location`, or `id`      | [`token`]      | The type of appointments that can be booked into this slot. Example: `12345`
-`schedule.actor`  | This and `slot-type`, or `id`                                    | [`reference`]  | A single or comma separated list of Practitioner references. Example: `Practitioner/12345`
-`-location`       | This and `slot-type`, or `id`                                    | [`reference`]  | A single or comma separated list of Location references. Example: `12345`
-`date`            | Yes when using `slot-type`                                       | [`date`]       | The Slot date-time. Example: `2016`
+`slot-type`       | Yes when using `schedule.actor` and\or `-location`, or `_id`     | [`token`]      | The type of appointments that can be booked into this slot. Example: `http://snomed.info/sct|394581000`
```

Comment:
Nit: backslash over forward slash seems weird to me

Reply:
Addressed with f54f352a457572e639496cf1f96efa62816cd041